itests/org.openhab.core.voice.tests/src/main/java/org/openhab/core/voice/internal/VoiceManagerImplTest.java

```diff
@@ -73,14 +67,17 @@ public void setUp() throws IOException {
         registerService(sink);
         registerService(voice);
 
-        Dictionary<String, Object> voiceConfig = new Hashtable<>();
-        voiceConfig.put(CONFIG_DEFAULT_TTS, ttsService.getId());
         ConfigurationAdmin configAdmin = super.getService(ConfigurationAdmin.class);
-        Configuration configuration = configAdmin.getConfiguration(VoiceManagerImpl.CONFIGURATION_PID);
-        configuration.update(voiceConfig);
+        Dictionary<String, Object> audioConfig = Hashtable.of("defaultSink", sink.getId());
```

Comment:
Sorry, the previous suggestion wasn't good since the IDE suggested `Map.of ` which is an interface implemented by Hashtable, so let's revert these:

```suggestion
        Dictionary<String, Object> audioConfig = new Hashtable<>();
        audioConfig.put("defaultSink", sink.getId());
```